projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -529,7 +529,7 @@ public List<JobDeploymentStatus> readV1CronJobDeploymentStatuses() {
               .listNamespacedCronJob(
                   namespace, null, null, null, null, null, null, null, null, null, null);
     } catch (ApiException e) {
-      log.warn("Failed to read k8s cron jobs: ", e);
+      log.warn("Failed to read k8s cron jobs: ", new KubernetesException("", e));
```

Comment:
why do you wrap it in a new exception?

Reply:
Because KuberneresException prints more details (it prints the actual error from the K8S API Server which the default exception does not :( )